infrastructure/quick-deploy/aws/parameters.tfvars

```diff
@@ -301,42 +301,100 @@ mq = {
 mongodb = {
   node_selector = { service = "state-database" }
   # Uncomment persistent_volume to enable persistence, comment to disable
-  #persistent_volume = {
-  #  storage_provisioner = "efs.csi.aws.com"
-  #  resources = {
-  #    requests = {
-  #      storage = "5Gi"
-  #    }
-  #  }
-  #}
+  # persistent_volume = {
+  #   storage_provisioner = "efs.csi.aws.com"
+  #   resources = {
+  #     requests = {
+  #       storage = "10Gi"
+  #     }
+  #   }
 
   # Uncomment to define custom resources for MongoDB pods, comment for default values
-  mongodb_resources = {
-    # limits = {
-    #   "cpu"               = 4
-    #   "memory"            = "8Gi"
-    #   "ephemeral-storage" = "20Gi"
-    # }
-    # requests = {
-    #   "cpu"               = 2
-    #   "memory"            = "4Gi"
-    #   "ephemeral-storage" = "2Gi"
-    # }
-  }
+  # mongodb_resources = {
+  #   limits = {
+  #     "cpu"               = 4
+  #     "memory"            = "8Gi"
+  #     "ephemeral-storage" = "20Gi"
+  #   }
+  #   requests = {
+  #     "cpu"               = 2
+  #     "memory"            = "4Gi"
+  #     "ephemeral-storage" = "2Gi"
+  #   }
+  # }
```

Comment:
I would remove all those comments. If you want to change these, you can look up the variables.tf, but as most of the time, I want the default values, there is no need to pollute the parameters.tfvars with all of those.